CHANGELOG.rst

```diff
@@ -8,6 +8,7 @@ This document describes changes between each past release.
 
 **New features**
 
+- Added details attribute to 404 errors.
```

Comment:
nitpick: add reference to original issue.

Also, this change belongs to the *protocol* section of changes, and requires a line in the API changelog (docs/api/index.rst)